src/bvers.f.in

```diff
@@ -0,0 +1,50 @@
+C> @file
```

Comment:
please do a `git mv` instead of `mv` and then `git add`.

Reply:
I did do that actually.  Not sure why it may not seem correct(?)